src/Cooked/MockChain/GenerateTx.hs

```diff
@@ -71,8 +65,10 @@ generateTx ::
   Map Api.TxOutRef Api.TxOut ->
   -- | validators present in our environment
   Map Script.ValidatorHash (Script.Versioned Script.Validator) ->
+  -- | The collateral inputs and associated collateral wallet
+  Maybe (Set Api.TxOutRef, Wallet) ->
```

Comment:
Same here. I actually don't mind the expanded version. I just advocate for consistency. Could be `Maybe (Set Api.TxOutRef, Wallet)`, `Maybe (Collaterals, Wallet)`, or `MCollaterals` everywhere, they would all seem fine to me.